web/frontend/legacy/lib/ui/content/export_modal.js

```diff
@@ -6,12 +6,13 @@ import urls from 'libs/urls';
 delegate(document, 'a.action.export', 'click', (e) => {
   e.preventDefault();
   const experimental = e.target.classList.contains('export-experimental');
+  const aws_lambda = e.target.classList.contains('export-lambda');
```

Comment:
I always narrate this practice of javascript with a "tuck tuck".

If you wanted to inject as a javascript variable, I'm doing a couple of those at https://github.com/harvard-lil/h2o/blob/a4885ccf918c0deca3bf0f217eb893eebcbf77e8/web/main/templates/base.html#L14 . It's probably just fine to keep it here though.

Reply:
Is "tuck tuck" like "tsk tsk" or is it something else?

Reply:
Like you're tucking the data into the dom.

Reply:
Ah, that is nice and tidy! I like it! A thought: this whole file is full of class list / data attribute retrievals. Would it be better to refactor the whole file at once.... maybe even porting it out of our legacy JS and into Vue? Happy to change now if you prefer.

Reply:
I think it makes sense to do it on another pass, perhaps when we add options to the export process.

Reply:
Cool. Thanks for reviewing! FYI, I am about to push one last commit, a tweak to setting handling subsequent to a convo with Ben.

Reply:
(pushed!)